event.go

```diff
@@ -383,6 +383,10 @@ func (e *Event) Pretty() (out string, ok bool) {
 		return fmt.Sprintf("[*] %s has quit (%s)", e.Source.Name, e.Trailing), true
 	}
 
+	if e.Command == INVITE {
+		return fmt.Sprintf("[*] invited to %s by %s", e.Trailing, e.Source.Name), true
```

Comment:
[RFC 2812](https://tools.ietf.org/html/rfc2812#section-3.2.7) states that:

> Only the user inviting and the user being invited will receive notification of the invitation.

Therefore I didn't extract the invited user name from the parameter array. I hope the output format is ok otherwise, let me know what you think.

Reply:
IRCv3 adds [invite-notify](https://ircv3.net/specs/extensions/invite-notify-3.2.html), which allows you to receive notifications when others do it too. So, we will probably want to fetch it from the params list. Something like `[*] invited to %s by %s` if target is self (compare name to `Client.GetNick()`), and `[*] %s invited to %s by %s` if it's someone else. Or just the latter if you're lazy. 🤷‍♂️ 

As for host, no need. If they join, the host will be picked up already.

Reply:
Decided to implement the latter since it is more consistent this way.